src/main/java/com/example/event_manager/model/BillingsSummary.java

```diff
@@ -7,10 +7,10 @@
 @Getter
 public class BillingsSummary {
 
-  private long notPaided;
-  private long paided;
-  private double coastOfAllPaided;
-  private double coastOfAllNotPaided;
+  private long notPaid;
```

Comment:
its enough to have just flag if billing was resolved or not, there will be never something like half-resolved billing.
Simple as:
- someone owns 5PLN
- someone took back 5PLN
Don't consider case when someone took back 4PLN when owning 5PLN

Reply:
Okay. At this moment I must rewrite Billing class for your proposed changes. I doing it on other branch.  Can you agree this changes now? In next pull request I will add flag to summary. I remember about change paided to paid, i do it soon